packages/sdk/src/__tests__/getGithubIssueUrl.test.ts

```diff
@@ -9,6 +9,8 @@ describe('getErrorMessageWithLink', () => {
       body: 'This is a body',
     })
 
-    expect(stripAnsi(message)).toMatchInlineSnapshot(`TEST_GITHUB_LINK`)
+    expect(stripAnsi(message)).toMatchInlineSnapshot(
+      `"https://github.com/prisma/prisma/issues/new?body=This+is+a+body&title=This+is+a+title&template=bug_report.md"`,
```

Comment:
What triggered this change?

Reply:
I'm gonna be honest, I don't recall what triggered this change. I'm actually surprised that this test would succeed earlier. Maybe when this specific test was in `@prisma/migrate` it was excluded from running? Or maybe the `getGithubIssueUrl` utility was mocked?

Reply:
`TEST_GITHUB_LINK` looks like something we would be the result of a string replacement on the snapshots, we have snapshotserializers somewhere that do this kind of thing. Maybe the URL is covered in the other pacakge, but not here.

Reply:
Yes here, https://github.com/prisma/prisma/blob/main/packages/sdk/src/utils/jestSnapshotSerializer.js#L43:L45

Though in this case it's better to have the URL I think.

Reply:
So, should I keep my edit as it is now?

Reply:
I think it's good as it is yes 👍🏼

Reply:
As the URL here is also a test URL (I hope), keeping it around is perfectly fine - no additional value in more sanitization here.